srcpkgs/hydrus/files/hydrus.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Version=1.0
+Name=Hydrus Client
+Comment=Danbooru-like image tagging and searching system for the desktop
+Exec=hydrus-client
+Icon=/usr/lib/hydrus/static/hydrus_non-transparent.png
+Terminal=false
+Type=Application
+Categories=Application;FileTools;Graphics;Network;
```

Comment:
Could you submit this upstream so we don't have to maintain this file?

Reply:
Yeah, i'll submit it and see if they accept.

Reply:
Go merged into upstream.
I created a symlink from /usr/lib to /usr/share/applications in the template.
The current desktop file is valid for void, i'll add patches if it should divert from our file structure